contracts/GuildBank.sol

```diff
@@ -10,8 +10,10 @@ contract GuildBank is Ownable {
 
```

Comment:
There's going to be some changes required here. I'm realizing the way we are storing the ERC20 tokens into an array isn't right, since we aren't checking if the token is already in the array. I think we will need to store a mapping and rely on offchain listeners to catch events to build the array. What do you think?